app/services/coronavirus_pages/timeline_entry_builder.rb

```diff
@@ -0,0 +1,29 @@
+class CoronavirusPages::TimelineEntryBuilder
+  def create_timeline_entries
+    if timeline_entries_from_yaml.any?
+      TimelineEntry.transaction do
+        coronavirus_page.timeline_entries.delete_all
+
+        timeline_entries_from_yaml.each do |entry|
```

Comment:
I think this will need to add the items in reverse order to get the positioning correct because of the callback. As per: https://github.com/alphagov/collections-publisher/pull/1253/files#diff-8b6d13a1a94024bdc57bf166e53622328267634808dbea57021a9e514a5b4ceeR71

Reply:
yeah, took a while to get my head round the ordering of things, sorry. This is sorted now.